pkg/api/controller.go

```diff
@@ -3105,6 +3105,7 @@ func (c *Controller) ForgotPassword(w http.ResponseWriter, r *http.Request, body
 	err := c.resetPasswordRequest(r.Context(), body.Email)
 	if err != nil {
 		c.Logger.WithError(err).WithField("email", body.Email).Debug("failed sending reset password email")
+		writeError(w, http.StatusBadRequest, "bad request")
```

Comment:
1. you write two responses
2. you don't want to return 400 for all these errors - just for invalid email format - not helping me understand which email exists or not

Reply:
Fixed